transaction-relayer/src/main.rs

```diff
@@ -58,23 +59,29 @@ static GLOBAL: Jemalloc = Jemalloc;
 #[derive(Parser, Debug)]
 #[clap(author, version, about, long_about = None)]
 struct Args {
-    /// Port to bind to advertise for TPU
-    /// NOTE: There is no longer a socket created at this port since UDP transaction receiving is
-    /// deprecated.
-    #[arg(long, env, default_value_t = 11_222)]
+    /// DEPRECATED, will be removed in a future release.
```

Comment:
can we use something to mark as deprecated? ie https://github.com/rust-lang/rust/issues/102777